app/services/admin/acas_certificate_search_service.rb

```diff
@@ -8,7 +8,7 @@ def self.call(form)
         return
       end
       base_url = ENV.fetch('ACAS_API_URL')
-      response = HTTParty.get("#{base_url}/certificates/#{form.number}", format: :json, headers: { 'EtUserId': form.current_admin_user.email, 'Accept' => 'application/json', 'Content-Type' => 'application/json' })
+      response = HTTParty.get("#{base_url}/certificates/#{form.number.gsub(/\s+/, '')}", format: :json, headers: { 'EtUserId': form.current_admin_user.email, 'Accept' => 'application/json', 'Content-Type' => 'application/json' })
```

Comment:
If the form.number should do any filtering, should this not be part of the form rather than the code using the form ?

Im not saying it definitely should be - but maybe it should depending on how easy that is ?